Domains/1-Common/DistributionSystems/DistributionSystems.ecschema.xml

```diff
@@ -18,6 +18,7 @@
     </ECCustomAttributes>
 
     <ECEntityClass typeName="DistributionSystem" modifier="none" displayLabel="Distribution System" description="A network designed to receive, store, maintain, distribute, or control the flow of a distribution media.">
+        <!-- Note: Should be considered abstract, but is concrete for legacy/compatibility reasons. -->

```

Comment:
could you remind me why did we decide to consider this abstract?

Reply:
nevermind, i've read abstract and thought deprecated. my bad.

Reply:
Refreshed Jonas' memory on this decision offline.